src/convertToObject.js

```diff
@@ -6,7 +6,16 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const arrayWithStyles = sourceString.split(';')
+    .map((el) => el.trim());
+  const resultObject = {};
+
+  for (const elem of arrayWithStyles) {
+    const pair = elem.split(':').map((el) => el.trim());
+    resultObject[pair[0]] = pair[1];
+  }
```

Comment:
The task requires avoiding loops. Consider using `reduce` to transform the array into an object without explicitly using a loop.